src/python_testing/conformance_support.py

```diff
@@ -151,6 +162,19 @@ def __str__(self):
         return f'{self.code}'
 
 
+class device_feature:
+    ''' This is different than element feature because device types use "features" that aren't reported anywhere'''
+
+    def __init__(self, feature: str):
+        self.feature = feature
+
+    def __call__(self, feature_map: uint = 0, attribute_list: list[uint] = [], all_command_list: list[uint] = []) -> ConformanceDecision:
+        return ConformanceDecision.OPTIONAL
+
+    def __str__(self):
+        return f'{self.feature}'
```

Comment:
self.feature is already a string, so 
```suggestion
        return self.feature
```
?
Or do we need a clone here? I guess same comment to existing code ... it seems it does the same.

Reply:
nah, it was just because the ones below used that style, so it ended up like this. Removed these in the next commit.